### PR TITLE
EZP-26011: Fixed the issue where the UDW `loadContent` flag is not taken into account

### PR DESCRIPTION
**Pull request by dpobel**
JIRA: https://jira.ez.no/browse/EZP-26011
Needed by https://github.com/ezsystems/PlatformUIBundle/pull/642
# Description

Make sure Content on the Fly takes the UDW `loadContent` flag into account.
# Tests

manual tests only

### CODE REVIEW

**Comment by andrerom** [comment]
Review ping @dew326 @yannickroger @lserwatka

**Comment by dew326** [comment]
+1

**Comment by lserwatka** [comment]
+1

File: bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js [hunk 1]
```diff
@@ -224,21 +224,27 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
...
          * Loads the content location of the published content
          *
-         * @method _setSelectedLocation
+         * @method _loadContentLocation
          * @protected
          * @param event {Object} event facade
          */
         _loadContentLocation: function (event) {
-            var host = this.get('host');
+            var udwService = this.get('udwService'),
```
Comment (dpobel) [inline bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js]: self note: `var udwService = this.get('host'),`

**Comment by yannickroger** [comment]
+1

**Comment by dpobel** [comment]
@andrerom how do we proceed in terms of branch/merge for this ?

**Comment by andrerom** [comment]
@dpobel Anyone in dev team can in practice do that, but anyone who has involved here including now yourself would be best candidates. As I have been involved I'll merge this one.

As for branch: Afaik don't see a need for branch here. 1. next version is done on platform ui master, 2. master here is 0.x versions, this fix typically 0.1.x update. ping me on skype if you had something else in mind here ;)

**Pull request merged**